src/main/java/org/betonquest/betonquest/database/PlayerData.java

```diff
@@ -31,11 +32,12 @@
  * Represents an object storing all profile-related data, which can load and save it.
  */
 @SuppressWarnings({"PMD.TooManyMethods", "PMD.CommentRequired"})
+@SuppressFBWarnings("JLM_JSR166_UTILCONCURRENT_MONITORENTER")
```

Comment:
Why was it required to add this now?

Reply:
not sure, but it is now an issue.

Sometimes i have the fealing that some issues appear, when related code is change, or even code that seems to be unrelated